bootcfg/api/handlers.go

```diff
@@ -25,3 +30,37 @@ func logRequests(next http.Handler) http.Handler {
 	}
 	return http.HandlerFunc(fn)
 }
+
+// selectGroup selects the Group whose selectors match the query parameters,
+// adds the Group to the ctx, and calls the next handler. The next handler
+// should handle a missing Group.
```

Comment:
hm, selectGroup is a HTTP handler that deals with query parameters, contexts, and http.Requests. The proto service SelectGroup is the underlying logic which is shared.

Maybe within the server package I'll leave comments in the proto, but in `api` (probably should be called `www` or `web`) they'd make sense?